client/app/scripts/superdesk/editor/editor.js

```diff
@@ -1225,6 +1267,18 @@ angular.module('superdesk.editor', ['superdesk.editor.spellcheck', 'angular-embe
                 }
             }
         };
-    }]);
+    }]).config(['embedlyServiceProvider', 'config', function(embedlyServiceProvider, config) {
+        // embed.ly private key
+        console.log(config);
```

Comment:
Oh, OK, then just don't forget to add the `WIP` label so that this does not get merged prematurely. :)